packages/liveblocks-react/test-d/augmentation.test-d.tsx

```diff
@@ -259,6 +259,14 @@ declare global {
   expectType<number>(room.getPresence().cursor.x);
   expectType<number>(room.getPresence().cursor.y);
   expectError(room.getPresence().nonexisting);
+
+  expectType<string>(classic.useRoom({ allowOutsideRoom: false }).id);
+  expectType<string | undefined>(
+    classic.useRoom({ allowOutsideRoom: true })?.id
+  );
+  expectType<string | undefined>(
+    classic.useRoom({ allowOutsideRoom: !Math.random() })?.id
```

Comment:
This will always be `false`, is it meant to be this? `Math.random() < 0.5`. Same in other places.